src/components/StepButtons.js

```diff
@@ -0,0 +1,95 @@
+import _ from 'lodash/fp'
+import { Fragment } from 'react'
+import { div, h, span } from 'react-hyperscript-helpers'
+import { Clickable } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import colors from 'src/libs/colors'
+
+
+export const params = {
+  buttonWidth: 170,
+  buttonHeight: 50,
+  dotSpace: 7,
+  dotSize: 6,
+  fontSize: 16
+}
+
+
+const styles = {
+  container: {
+    display: 'flex', alignItems: 'flex-start',
+    margin: '2rem 0'
+  },
+  button: isActive => ({
+    display: 'flex', alignItems: 'center',
+    flex: 'none',
+    width: params.buttonWidth, height: params.buttonHeight,
+    borderRadius: params.buttonHeight/2, borderWidth: 2, borderStyle: 'solid',
```

Comment:
Is this supposed to be `params.buttonHeight/2` or `params.buttonHeight: 2`?

Reply:
I thii

Reply:
I think spaces around arithmetic operators might make it clearer that the intent is the former.